nunaliit2-couch-sdk/src/main/templates/config/multimedia.properties

```diff
@@ -43,7 +43,7 @@
 # -f mp4 Force input or output file format.
 # %2$s Specifies output file
 #
-#ffmpegConvertVideoCommand=avconv -i %1$s -y -acodec libvo_aacenc -ab 48000 -ac 2 -vcodec libx264 -b:v 128000 -r 24 -vf scale=320:-2 -threads 0 -f mp4 %2$s
+#ffmpegConvertVideoCommand=ffmpeg -i %1$s -y -acodec libvo_aacenc -ab 48000 -ac 2 -vcodec libx264 -b:v 128000 -r 24 -vf scale=320:-2 -threads 0 -f mp4 %2$s
```

Comment:
I can't find an -ab option in ffmpeg documentation (or in avconv). I suspect -ar may be what we want.

Reply:
We have another activity underway with an atlas where we are revisiting the ffmpeg parameters in light of newer codecs/optimization/etc. If we need to address this, we can do it then. I'm also wary of introducting a colon to the command-line arguments without some serious testing. I can imagine it tripping us up. Thanks though.